examples/fit_ODNP_data.py

```diff
@@ -0,0 +1,105 @@
+"""Fitting ODNP Datasets for Ksigma
+===================================
+The T1(p) and E(p) integrals are pulled from an H5 file and are fit to extract the cross relaxivity of the sample. The R1(p) is first interpolated by fitting the self relaxivity to a polynomial. This fit is then used to fit the linear regime of the enhancement data. The cross relaxivity is then calculated using the output of these fits as well as the sample parameters that are fed to it (e.g., ppt value, and concentration).
+"""
+from pyspecdata import *
+from sympy import symbols, Symbol, latex
+
+target_directory = os.path.normpath(getDATADIR("AG_processed_data"))
+h5_folder = "ras.h5"
+filename = "220616_E37"
+# {{{Dataset Parameters
+SL_conc_M = 455.2223e-6
+ppt = 1.5167e-3
```

Comment:
Why don't these come from a file?

Reply:
These are saved in the acquired dataset.h5 file. Here we are pulling the table of integrals h5 file. I can load the acquired dataset to pull the acquisition parameters if you think that'd be better. I think that would overcomplicate but you know best

Reply:
Isn't the easier solution to use set_prop on the table of integrals and transfer them?

Reply:
fixed